scripts/BuildAndTest.ps1

```diff
@@ -195,13 +190,6 @@ if (-not $NoClean) {
 
 Install-VersionConstantsFile
 
-if (-not $NoRestore) {
```

Comment:
This didn't actually stop the subsequent dotnet invocations from restoring on this nuget.exe restore we no longer need. I could make the switch work properly, but I figured we don't need it.